basic/aws_sso_strategy/aws_sso_strategy.tf

```diff
@@ -26,7 +26,7 @@ module "sso_connector" {
     aws = aws.sso
   }
 
-  environment       = "main"
+  environment = local.environment_name
```

Comment:
"main" is also hard-coded in other places in this file (e.g. strategy.name, integration.name) -- do we want to replace them all?

Reply:
Ah, right I changed this because I was testing locally. Let me look at the other examples to see what we do there

Reply:
I'll leave it as `main` for now, since that's how the AWS IAM connector and lambda connector are defined